code/game/machinery/computer/arcade.dm

```diff
@@ -872,7 +896,7 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	add_fingerprint(usr)
 	updateUsrDialog()
 	busy = FALSE
-	return
+	usr?.mind.adjust_experience(/datum/skill/gaming, xp_gained+1)
```

Comment:
do not use the cursed usr var

Reply:
oh wait this is acceptable

Reply:
What is the cursed usr var?

Reply:
I believe it refers to the player that did _something_ that ended up calling the proc. It is well known to be anything but consistent in the vast majority of cases.